src/main/java/com/example/demo/rest/TaskResource.java

```diff
@@ -57,7 +58,65 @@ public ResponseEntity<?> getUserTasks(@PathVariable("userId") int userId, @Reque
         user.get().setTasks(tasks);
         return new ResponseEntity<>(user.get().getUserWithTasks(), HttpStatus.OK);
     }
+    @GetMapping("/managerId")
```

Comment:
Can we change this to "/api/v1/users/{userId}/teamtasks" and get the managerId from the provided userId?
Extract to a different Resource class if needed.